.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "src/3rdparty/libcrashreporter-qt"]
 	path = src/3rdparty/libcrashreporter-qt
 	url = git://github.com/guruz/libcrashreporter-qt.git
+[submodule "doc/_shared_assets"]
+	path = doc/_shared_assets
+	url = git@github.com:owncloud/docs-themes.git
```

Comment:
I suggest to name the path exactly as the the repo.
How about path = docs-themes or doc/docs-themes ?

Reply:
Please use `git://github.com/owncloud/docs-themes.git`, otherwise the builds will fail because we are not rewriting the GitHub submodule URLs.

Reply:
I kept the path as close to the original as possible for simple continuity.

Reply:
@jnweiger, fair enough, I guess the closer the name the better.

Reply:
@settermjd for git submodule rename/removal keep https://stackoverflow.com/a/1260982/2172072 at hand for future reference.